minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneClusterControl.java

```diff
@@ -123,11 +114,12 @@ public int exec(Class<?> clz, String[] args) throws IOException {
 
   public Entry<Integer,String> execMapreduceWithStdout(Class<?> clz, String[] args) throws IOException {
     String host = "localhost";
-    String[] cmd = new String[3 + args.length];
-    cmd[0] = getToolPath();
-    cmd[1] = getJarFromClass(clz);
-    cmd[2] = clz.getName();
-    for (int i = 0, j = 3; i < args.length; i++, j++) {
+    String[] cmd = new String[4 + args.length];
+    cmd[0] = getAccumuloUtilPath();
+    cmd[1] = "hadoop-jar";
+    cmd[2] = getJarFromClass(clz);
+    cmd[3] = clz.getName();
```

Comment:
Okie doke. Half of this was me wondering how it fails when the configuration is actually wrong (e.g. we can't find the hadoop installation/jars), but that isn't related to this changeset.